### PR TITLE
allow the configuration of serveraliases

### PR DESCRIPTION
**Pull request by igalic**
In some environments, foreman might have a different name to different
networks. Different implementations of SSL (jruby, curl-tls, openssl),
appear to have different levels of acceptance for such a certificate
(mis)configuration in Apache httpd's vhost.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -24,6 +24,8 @@
 #
 # $servername::                	Server name of the VirtualHost in the webserver
 #
+# $serveraliases::             	Server aliases of the VirtualHost in the webserver
```
Comment (ekohl) [inline manifests/init.pp]: I think it could use ```type:array``` and a ```validate_array``` in the body.

**Comment by ekohl** [comment]
Looking good.

**Comment by mmoll** [comment]
tests fail :panda_face:

**Comment by igalic** [comment]
:( worked on my machine :(

**Comment by mmoll** [comment]
strange. when moving the validate_array to init.pp, it works

**Comment by mmoll** [comment]
also, please rebase :)

**Comment by ekohl** [comment]
> strange. when moving the validate_array to init.pp, it works

That's because in our tests we have no pre condition on ```foreman``` so the value is ```undef```. You need to set it to something even in our tests.

**Comment by mmoll** [comment]
see #401

**Pull request closed without merging**